roles/git_repo/tasks/branch.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: "Create local repository branch tracking specified remote branch"
+  ansible.builtin.command:
```

Comment:
I've tried to use the git_config module for this sort of thing, which was suggested by the maintainers of the git module as that module doesn't support everything that we need to do here, but branch creation was the one area where I could never git_config to work.

Doing it with the command module smells somewhat hacky, but it's not only worked very reliably, it also ends up being quite a bit simpler.